locations/alpineskyline.json

```diff
@@ -429,7 +436,7 @@
       {
         "name": "The Twilight Bell (Rift)",
         "access_rules": [
-          "@Alpine Logic/Can access Time Rift - The Twilight Bell"
+          "@Alpine Logic/Can access Time Rift - The Twilight Bell, @Alpine Logic/Can access Free Roam"
```

Comment:
The time rift entrance itself is in `locations/entrances/chapter_time_rifts.json` under `Chapter Time Rift Entrances/Chapter4/TimeRift_Water_Alp_Goats`. Because you must complete The Twilight Bell to spawn the Time Rift entrance and if you can complete The Twilight Bell, you must be able to reach the Time Rift entrance, the entrance only checks for having complete The Twilight Bell.